en/blazor-support.md

```diff
@@ -52,12 +52,35 @@ Generating Blazor application has the following limitations:
 - Bootstrap is turned off in the generated application for now.
 
 Generated components have these limitations:
-- Navigation drawer works only in the overlay state. The mini version does not work too.
-- Date picker works only in dropdown mode.
-- The input group does not support date type.
-- Radio group does not support horizontal mode.
-- Grid cannot generate a toolbar. This disallows the generation of advanced filtering, exporting, pinning, and hiding. Setting cell and row selection do not work simultaneously. Column selection, paging, grouping, and column moving are not supported yet. Filtering is working only in excel style mode.
-- For all components toggle actions are not generated yet.
+- Card:
+    - cannot generate reversed actions.
+    - cannot generate divider in actions column card type.
+- Navigation Bar - cannot generate title.
+- Navigation Drawer - does not support pin state and pin threshold.
+- Avatar:
+    - initials type shows entire contend instead of first two chars.

```

Comment:
👀 How is that relevant to the code generation? Sounds like a component issue to me, if one at all, might be intended too